src/apps/frontend/components/input/input.component.tsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+import { Input as BaseInput, InputProps as BaseInputProps } from 'baseui/input';
+import React from 'react';
+
+export interface InputProps extends BaseInputProps {
+  testId: string;
+  value?: string;
+  placeholder?: string;
+}
+
+const Input: React.FC<InputProps> = ({
+  value,
+  placeholder,
+  testId,
+  ...otherProps
+}) => (
+  <BaseInput
+    {...otherProps}
+    overrides={{
+      Input: {
+        props: {
+          'data-testid': testId,
+        },
+        style: ({ $theme }) => ({
+          backgroundColor: `${$theme.colors.inputBackground}`,
+          color: `${$theme.colors.fontColor}`,
+          caretColor: `${$theme.colors.fontColor}`,
+        }),
```

Comment:
Why do we need to override this? Can we not and use the form control directly at consumer?

Reply:
Done